Set up skeleton of models needed

helin24 opened the pull request:
- Add php script to manage connecting and disconnecting
- Fill in most of LDAP model

helin24 committed:
Set up skeleton of models needed

- Add php script to manage connecting and disconnecting
- Fill in most of LDAP model

sc250024 commented on openvpn.php:
```diff
@@ -0,0 +1,19 @@
+require_once('models/ConnectionManager.php');
+
+list($script, $action, $user) = $argv;
+// $argv[0] is script location
```
Not sure if the location of the script is needed since the BASH script will actually handle that part; `$argv[0]` should be the location of the tmp file that OpenVPN generates

sc250024 committed:
BASH scripts called from OpenVPN process

sc250024 committed:
Quick change to disconnect script

helin24 committed:
Add iptables statements for adding/deleting rules (untested)

helin24 committed:
Complete logic to extract addresses from LDAP

- Modify bash script to connect to php scripts
- Fill in getNetwork function in LDAP class
- Modify address class

sc250024 committed:
Please work

helin24 committed:
remove line

helin24 committed:
Return LDAP rules as objects

helin24 committed:
Throw exception if no rules and pass user IP

helin24 committed:
Update iptables rules, use more env vars

helin24 committed:
Fix deletion and insertion of rules

helin24 committed:
Flush and delete chain only if it exists

helin24 committed:
Create LDAP result if no subentries

helin24 committed:
Add more iptables rules and add docs

helin24 committed:
Add more iptables rules

helin24 committed:
Remove additional nat rules

helin24 committed:
Add in-interface

helin24 committed:
Fix using inInterface

helin24 committed:
Push routes to client

helin24 committed:
Fix disconnect script

helin24 committed:
Access environment variables from php

helin24 committed:
Correct file variable and use relative path
